app/helpers/chart_helper.rb

```diff
@@ -0,0 +1,17 @@
+module ChartHelper
```

Comment:
IMO we should strive for this to be under a `patternfly-rails` gem that provides helpers for Patternfly common actions, under the Patternfly organization